schemas.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel, validator
+
+#inputi type validation
+class MultipleJokesRequestParams(BaseModel):
+    count: int
+
+    #in range validation
```

Comment:
I would say you don't need this comment. This method name `is_in_range` is clear enough :)